tools-v2/pkg/config/curve.yaml

```diff
@@ -20,6 +20,8 @@ curvebs:
   mdsAddr: 127.0.0.1:6700,127.0.0.1:6701,127.0.0.1:6702   # __CURVEADM_TEMPLATE__ ${cluster_mds_addr} __CURVEADM_TEMPLATE__
   mdsDummyAddr: 127.0.0.1:7700,127.0.0.1:7701,127.0.0.1:7702  # __CURVEADM_TEMPLATE__ ${cluster_mds_dummy_addr} __CURVEADM_TEMPLATE__
   etcdAddr: 127.0.0.1:23790,127.0.0.1:23791,127.0.0.1:23792  # __CURVEADM_TEMPLATE__ ${cluster_etcd_addr} __CURVEADM_TEMPLATE__
+  snapshotAddr: 192.168.20.149:5550,192.168.20.149:5551,192.168.20.149:5552
```

Comment:
add suffix: `# __CURVEADM_TEMPLATE__ ${cluster_etcd_addr} __CURVEADM_TEMPLATE__`
it's used by curveadm.

Reply:
it's better `s/192.168.20.149/127.0.0.1/`

Reply:
done